packages/lib-classifier/src/plugins/drawingTools/components/index.js

```diff
@@ -4,3 +4,4 @@ export { default as DeleteButton } from './DeleteButton'
 export { default as DrawingToolRoot } from './DrawingToolRoot'
 export { default as Line } from './Line'
 export { default as Point } from './Point'
+export { default as TranscriptionLine } from '@plugins/drawingTools/experimental/components/TranscriptionLine'
```

Comment:
I dislike looking up through the filesystem for dependencies, but that's mostly when the path is something like `../../../target`. My preference is to use aliases instead of `../` wherever possible but I have no strong opinion here.

Reply:
I think what I really want here is something like
```js
import * as experimentalComponents from '@plugins/drawingTools/experimental/components'
export { ...experimentalComponents }
```

Reply:
Meant to write before hitting resolve: I don't have a strong preference. I'm still getting used to the aliased imports.